src/server/rest/router/GlobalRouter.ts

```diff
@@ -7,14 +7,17 @@ import { utilRouter } from './util/UtilRouter';
 
 export const globalRouter = express.Router();
 
-// API
-globalRouter.use('/v1/api', AuthService.authenticate(), tenantRouter);
+// API version 1 routes
+const currentVersion = 1;
+const currentVersionBaseURI = `/v${currentVersion}`;
```

Comment:
If we add a directory v1, no need to add a variable here I think

Reply:
You will not have the directory name mapping into the router out of box, unless you do some directory listing and parsing. It's better to just use a constant.